sig-cli/charter.md

```diff
@@ -0,0 +1,50 @@
+# SIG CLI Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [sig-governance].
+
+## Scope
+
+The Command Line Interface SIG (SIG CLI) is responsible for kubectl and
+related tools. This group focuses on command line tools and
+libraries to interface with Kubernetes API's.
+
+### In scope
+
+SIG CLI section in [sigs.yaml]
+
+#### Code, Binaries and Services
+
+SIG CLI code include command line tools and binaries for working with Kubernetes
+API's. Examples of these binaries include: kubectl and kustomize.
+
+### Out of scope
```

Comment:
would it make sense to include some "Cross-cutting and Externally Facing Processes" aspects of how sig-cli collaborates with the sigs that own other binaries to improve command consistency?

Reply:
Not quite sure what you mean by that?